### PR TITLE
Feature login and registration components

### LINKED ISSUE #48: Login and registration

### PR DESCRIPTION
**Pull request by veca22**

### CODE REVIEW

File: sbag/generators/javascript/entities/shared/auth/login/login.component.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<div class="d-flex p-2 justify-content-center">
+	<form [formGroup]="loginForm" class="col-sm-4" (ngSubmit)="onSubmit(loginForm.value)">
+		<div class="form-group">
+			<label for="email">Email</label>
```
Comment (balsa-sarenac) [inline sbag/generators/javascript/entities/shared/auth/login/login.component.html]: Username is used for login

File: sbag/generators/javascript/entities/shared/auth/register/register.component.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  selector: 'app-register',
+  templateUrl: './register.component.html',
+  styleUrls: ['./register.component.css']
+})
+export class RegisterComponent implements OnInit {
+
+    registerForm: any;
+    constructor(private formBuilder: FormBuilder) {}
+    ngOnInit(): void {
+      this.registerForm = this.formBuilder.group({
```
Comment (balsa-sarenac) [inline sbag/generators/javascript/entities/shared/auth/register/register.component.ts:14]: Add username here as well

**Review by balsa-sarenac — CHANGES_REQUESTED** [review]

File: sbag/generators/javascript/entities/shared/auth/register/register.component.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<div class="d-flex p-2 justify-content-center">
+	<form [formGroup]="registerForm" class="col-sm-4" (ngSubmit)="onSubmit(registerForm.value)">
+		<div class="form-group">
+			<label for="username">Username</label>
+			<input class="form-control" required="required" type="text" id="username" formControlName="username" />
+		</div>
```
Comment (balsa-sarenac) [inline sbag/generators/javascript/entities/shared/auth/register/register.component.html:10]: Add email for registration

File: sbag/generators/javascript/entities/shared/auth/register/register.component.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        email: '',
+        username: '',
+      });
```
Comment (balsa-sarenac) [inline sbag/generators/javascript/entities/shared/auth/register/register.component.ts:18]: Password is missing?

**Review by balsa-sarenac — CHANGES_REQUESTED** [review]

**Review by nikolina-sarenac — APPROVED** [review]

**Review by vtanja — APPROVED** [review]

**Review by balsa-sarenac — APPROVED** [review]

**Pull request merged**